amgut/db/patches/0017.sql

```diff
@@ -0,0 +1,195 @@
+-- August 17, 2015
+-- Rearrange barcodes to be attached to the correct projects
+Do $do$
+DECLARE
+    ag_pid bigint;
+    pid bigint;
+BEGIN
+ag_pid := (SELECT project_id FROM barcodes.project WHERE project = 'American Gut Project');
+
+-- Assign british gut kits to british gut and american gut
+CREATE TEMP TABLE hold_table (
+    barcode varchar,
+    project_id bigint
+);
+
+pid := (SELECT project_id FROM barcodes.project WHERE project = 'British Gut Project');
```

Comment:
so why do `ag_pid`? what's the reason to be concerned about namespace declarations, aren't you more concerned about the potential namespace collisions?